scopes/preview/preview/preview.docs.mdx

```diff
@@ -1,5 +1,5 @@
 ---
-description: Handles the bundlings and rendering of component compositions and documentations
+description: Provides a visually rich Component preview, in an isolated browser-based environment.
```

Comment:
Previews components in an isolated, browser-based environment.

Reply:
done